server/chat-formatter.ts

```diff
@@ -458,11 +441,7 @@ class TextFormatter {
 
 /**
  * Takes a string and converts it to HTML by replacing standard chat formatting with the appropriate HTML tags.
- *
- * @param {string} str
  */
-function formatText(str, isTrusted = false) {
+export function formatText(str: string, isTrusted: boolean = false) {
```

Comment:
The compiler should be able to infer the `boolean`
```suggestion
export function formatText(str: string, isTrusted = false) {
```